dashboard/config/environments/test.rb

```diff
@@ -29,6 +29,10 @@
 
     # Version of your assets, change this if you want to expire all your assets.
     config.assets.version = '1.0'
+
+    # Speeds up the env loading by avoid loading all i18n files up front.
```

Comment:
Grammar nit: technically, it's more correct to use "avoiding" here:

```suggestion
    # Speeds up the env loading by avoiding loading all i18n files up front.
```

But "loading by avoiding loading" is a bit of a mouthful. What about something like,

```suggestion
    # Avoid loading all i18n files up front, which can significantly slow down initialization.
```